addons/website_slides/controllers/main.py

```diff
@@ -62,14 +62,15 @@ def _fetch_slide(self, slide_id):
         except AccessError:
             return {'error': 'slide_access'}
         return {'slide': slide}
+    a = dict()
```

Comment:
What is this dict ?

Reply:
è§!zè!§z wasting time of everyone; obviously not wanted ^^